<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Roman" failures="1" tests="2" name="Error Validations Test" time="61.154" errors="0" timestamp="2024-03-26T14:04:02 EDT">
  <testcase classname="romangaznavi.tests.ErrorValidationsTest" name="loginErrorValidation" time="1.182"/>
  <testcase classname="romangaznavi.tests.ErrorValidationsTest" name="productErrorValidation" time="15.211">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at romangaznavi.tests.ErrorValidationsTest.productErrorValidation(ErrorValidationsTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- Error Validations Test -->
